aiden/class2-2/10845.py

```diff
@@ -0,0 +1,43 @@
+import sys
+from collections import deque
+
+input = sys.stdin.read
+data = input().split()
+n = int(data[0])
+queue = deque()
+
+index = 1
+for i in range(n):
+    command = data[index]
```

Comment:
```suggestion
for command in data[1:]:
```

같이 쓰면 조금 더 코드 관리가 편할것같습니다!

Reply:
index의 역할이 [push,pop,size,empty,front,back] 을 지정해주는 요소라서 data[1:]으로 작성을 하면 섞여서 어려울 것 같다고 생각이듭니다. 
![설정](https://github.com/user-attachments/assets/9f9d5346-acb2-4a3c-af34-b52f1d192b57)
해당 부분에서 문제에서의 예제 입력1을 입력 후 command를 출력 시 push만 출력이 되는데 
![image](https://github.com/user-attachments/assets/4b8a216b-051e-4c80-afc4-0eb9e3bfc987)
제안해주신 방법으로 출력시 위와 같이 출력이 되어 command를 출력할 때 조금 더 어려움이 있을 것 같습니다

Reply:
아 데이터를 전부 한줄로 세우고 2개 들어오는 경우에 `index += 2` 하고있군요
제가 잘못 생각했네요 ㅎㅎ
또 하나 배워갑니다